tools/build_travis.py

```diff
@@ -137,6 +137,30 @@
                "rtos" : ["RTOS_1", "RTOS_2", "RTOS_3"],
                "usb"  : ["USB_1", "USB_2" ,"USB_3"],
                }
+     },
+    {"target": "K64F",
+     "toolchains": "GCC_ARM",
+     "tests": {""     : ["MBED_2", "MBED_10", "MBED_11", "MBED_15", "MBED_16", "MBED_17"],
+               "fat"  : ["MBED_A12", "MBED_19", "PERF_1", "PERF_2", "PERF_3"],
```

Comment:
MBED_A19 is located here: https://github.com/ARMmbed/mbed-os/blob/master/features/unsupported/tests/mbed/i2c_eeprom/main.cpp

You can lookup where these tests are in the file here: https://github.com/ARMmbed/mbed-os/blob/master/tools/tests.py

For instance, this is the entry for MBED_A19: https://github.com/ARMmbed/mbed-os/blob/master/tools/tests.py#L265